src/Workspaces/CoreTest/CodeCleanup/NormalizeModifiersOrOperatorsTests.cs

```diff
@@ -1049,6 +1049,26 @@ End Class
             Verify(code, expected);
         }
 
+        [Fact]
+        [WorkItem(1534, "https://github.com/dotnet/roslyn/issues/1534")]
+        [Trait(Traits.Feature, Traits.Features.NormalizeModifiersOrOperators)]
+        public void TestColonEqualsToken()
+        {
+            var code = @"[|Module Program
+    Sub Main(args As String())
+        Main(args:     =args)
+    End Sub
```

Comment:
@Pilchie The space between 'args' and ':' gets handled by the formatter because 'args' and ':=' are tokens. This was different because this is handled by the CodeCleanupService. Nevertheless, I will update the test case to prevent regression.